### PR TITLE
APPS-2685 (Nice-to-have) Print NF workdir at beginning of job log

### PR DESCRIPTION
**Pull request by r-i-v-a**
- [x] Merge other branch this branch was based on first

Testing details on Jira --> APPS-2685

### CODE REVIEW

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh
Comment (r-i-v-a) [inline src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:463]: Edited wording and order

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh
Comment (r-i-v-a) [inline src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:461]: This is now covered by printing "NF workdir..." always

Comment (yihchii) [reply src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:461]: Would this be a breaking change to current users? For example, what if users rely on reading/parsing the existing log format? 
What is the down side of keeping the existing log but adding new ones?

Comment (yihchii) [reply src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:461]: Please address the command. Once it is resolved, this LGTM. Thanks!

**Review by yihchii — APPROVED** [review]
Please review the review (I am concerned about if i'd be breaking changes) before merge.

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh
Comment (r-i-v-a) [reply src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:461]: I reverted the other changes besides adding the "=== NF workdir      : ${NXF_WORK}" line.

**Comment by r-i-v-a** [comment]
Please review, @mhrvol @YuxinShi0423 (review needed from someone with write access).

**Review by YuxinShi0423 — APPROVED** [review]
Looks good to me :)

**Pull request merged**
